tools/encodings/arabic_unique-glyphs.nam

```diff
@@ -79,74 +79,29 @@
 0x0670 # ARABIC LETTER SUPERSCRIPT ALEF
 0x0671 # ARABIC LETTER ALEF WASLA
 0x0679 # ARABIC LETTER TTEH
-0x067A # ARABIC LETTER TTEHEH
```

Comment:
@khaledhosny I'm curious what you think about which unicode characters should be included in the default arabic subset, and what the effect of not including these characters is? :)